sdformat_urdf/src/sdformat_urdf.cpp

```diff
@@ -493,12 +493,16 @@ sdformat_urdf::convert_joint(const sdf::Joint & sdf_joint, sdf::Errors & errors)
     case sdf::JointType::PRISMATIC:
       urdf_joint->type = urdf::Joint::PRISMATIC;
       break;
+    case sdf::JointType::UNIVERSAL:
+      urdf_joint->type = urdf::Joint::FLOATING;
+      break;
+    case sdf::JointType::BALL:
+      urdf_joint->type = urdf::Joint::FLOATING;
+      break;
     case sdf::JointType::INVALID:     // Unsupported: fall through to default
-    case sdf::JointType::BALL:        //  |
     case sdf::JointType::GEARBOX:     //  |
```

Comment:
I'd recommend making all joints types in SDFormat that don't have URDF equivalents map to `urdf::Joint::FLOATING` (except `sdf::JointType::INVALID`). That way other models with those joints could still be used as long as someone was willing to write a custom TF publisher for them.